### PR TITLE
feat(env): add command to list/get/set/import environment variables 

### LINKED ISSUE #1011: Add CLI command to get/set/list environment variables
This was requested in [Netlify community](https://community.netlify.com/t/are-envs-set-in-build-plugins-available-during-function-runtime/18842/12).

Users can currently get/set/list environment variables using `netlify api updateSite` and `netlify api getSite` but this is quite cumbersome and also:
  - `updateSite` replaces all environment variables (set in the UI), instead of appending to them
  - `getSite` only returns the environment variables set in the UI

It would be nice for users to be able to do something similar to `heroku config:set ENV=value`:
  - `netlify env list`: print all environment variables. This would include the ones set in `netlify.toml`, including `contexts.*.environment` (this can easily be done by using the return value of `@netlify/config`)
  - `netlify env get envVarName`: print a single environment variable's value
  - `netlify env set envVarName value`: this would set the value in the UI, not in `netlify.toml`
  - `netlify env delete envVarName`: or alternatively, allow doing `netlify env set envVarName ""`

What do you think?

### PR DESCRIPTION
**Pull request by Pyrax**
**- Summary**

Closes #1011

This PR introduces commands for managing environment variables from the CLI. It uses the Netlify API to set environment variables for the current site in the UI. 

| Command | Description |
| -------------- | --------------- |
| `netlify env:list` | Shows a table of env variables including those set in `netlify.toml`.  |
| `netlify env:get NAME` | Prints the value of a specified variable including those set in `netlify.toml`. |
| `netlify env:set NAME VALUE` | Sets an environment variable in the UI. |
| `netlify env:set NAME` | Removes an environment variable in the UI. |
| `netlify env:import FILE_NAME` | Imports environment variables from `.env` files and sets them in the UI. |
| `netlify env:import --replaceExisting FILE_NAME` | Imports environment variables from `.env` files while deleting any previously set variables in the UI. Can be useful for e.g. renaming variables. |

For the manipulation commands `netlify env:import` and `netlify env:set` with the `--json` flag it returns all variables which are afterwards present for the site in the UI.

**- Test plan**

Automated tests are available in `/tests/command.env.test.js`. They are run serial because they create a test site on Netlify for making use of the API. This is similar to what is done in the tests for addons. Therefore, I also moved two functions into `/tests/utils` for re-use.

Test execution (total execution time ~ 30s): 
```
npx ava --verbose command.env.test.js

creating new site for tests: netlify-test-env-ocanhpby
  ✔ env:list --json should return empty object if no vars set (2.1s)
  ✔ env:get --json should return empty object if var not set (2.1s)
  ✔ env:set --json should create and return new var (2.5s)
  ✔ env:set --json should update existing var (2.3s)
  ✔ env:get --json should return value of existing var (2.2s)
  ✔ env:import should throw error if file not exists (2s)
  ✔ env:import --json should import new vars and override existing vars (2.4s)
  ✔ env:get --json should return value of var from netlify.toml (2s)
  ✔ env:list --json should return list of vars with netlify.toml taking priority (2s)
  ✔ env:set --json should unset var if value is set empty (2.3s)
  ✔ env:import --json --replace-existing should replace all existing vars and return imported (2.5s)
Performing cleanup
deleting test site "netlify-test-env-ocanhpby". 9189194a-dfaf-4e3f-8e0d-9c9880035690

  11 tests passed
```

**- Description for the changelog**

feat(env): add command to list/get/set/import environment variables 

**- A picture of a cute animal (not mandatory but encouraged)**

IMG_2019_AOJQW0312KMA

### CODE REVIEW

File: src/commands/env/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+EnvCommand.description = `Control environment variables for the current site`
+EnvCommand.examples = [
+  'netlify env:list',
+  'netlify env:get VAR_NAME',
+  'netlify env:set VAR_NAME value',
+  'netlify env:delete VAR_NAME',
+  'netlify env:import fileName',
+  // 'netlify env:export file-name'
```
[the remark this answers is not in the document]
Comment (Pyrax) [reply src/commands/env/index.js:31]: Oh yes, thanks!
(I decided against an export command as `netlify env:list --json` should do the job.)

File: src/commands/env/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    })
+  }
+}
+
+EnvCommand.description = `Control environment variables for the current site`
+EnvCommand.examples = [
+  'netlify env:list',
+  'netlify env:get VAR_NAME',
+  'netlify env:set VAR_NAME value',
+  'netlify env:delete VAR_NAME',
```
[the remark this answers is not in the document]
Comment (Pyrax) [reply src/commands/env/index.js:29]: You are right. I am going to add one and modify the `set` command.

**Comment by Pyrax** [comment]
- `netlify env:set` now allows empty values to be set to. A new test case verifies this.
- Added a command to delete variables which is called `netlify env:unset NAME` (unset should be familiar to bash users) with aliases of `netlify env:remove` and `netlify env:delete`.

**Review by erezrokah — APPROVED** [review]
@Pyrax, thank the quick follow up.

I added two more commits:
1. renamed a test.
2. Marked this as a beta feature (similar to https://cli.netlify.com/commands/build/#build), so we can gather some feedback before committing on the API.

If you're good with those changes I think this is ready to be merged.

**Comment by Pyrax** [comment]
Perfect, I'm good with it.

**Comment by erezrokah** [comment]
Posted this in our community, thank again @Pyrax!
https://community.netlify.com/t/cli-new-command-to-manage-env-variables/21893

**Pull request merged**